src/toggle/toggle.component.ts

```diff
@@ -159,6 +160,14 @@ export class Toggle extends Checkbox {
 		Toggle.toggleCount++;
 	}
 
+	getOffText() {
```

Comment:
getOffTextSubject perhaps? It looks confusing with the same name getters there.
same for getOnText
same for things in Table

Reply:
On one hand I don't disagree, on the other if I type it to `getOffText(): Observable<string>` that may be more explicit and accomplish the same idea ... If we change in the future we won't have to rename all the methods